### PR TITLE
[CET-9978] Error when viewing Bird's Eye Report in Backstage

### PR DESCRIPTION
**Pull request by PavelZurek**
## Change description

[CET-9978] Error when viewing Bird's Eye Report in Backstage

Bird's Eye Report trowed an error when attempting to display heatmap with archived entity.

## Type of change

- [x] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Improvement (improves codebase without changing functionality)

### Development

- [ ] The changelog has been updated as appropriate
- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

[CET-9978]: https://cortex1.atlassian.net/browse/CET-9978?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by igorRoog — DISMISSED** [review]
LGTM ✅

**Comment by igorRoog** [comment]
Could you maybe extract it to some utils and write some unit tests?

**Review by igorRoog — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.11.4
+
+- Fix Bird's Eye Report throwing error when it included archived entities
```
Comment (cristinabuenahora) [inline CHANGELOG.md:5]: Why do we have archived entities in the Bird's eye report?

Comment (PavelZurek) [reply CHANGELOG.md:5]: We don't include those in report, only the IDs are in the hierarchy response

**Pull request merged**